pandas/core/arrays/base.py

```diff
@@ -403,13 +404,13 @@ def argsort(self, ascending=True, kind='quicksort', *args, **kwargs):
         """
         # Implementor note: You have two places to override the behavior of
```

Comment:
can you augment the doc-string for argsort / _values_for_argsort to say that NaNs are placed at the end.